doc/release/launcher.rst

```diff
@@ -91,122 +94,309 @@ installation-specific configuration, e.g.), you can often use the ``-v``
 flag to capture the commands that the launcher executes on your behalf
 and customize them for your needs.
 
-
---------------------------------
 Forwarding Environment Variables
---------------------------------
+++++++++++++++++++++++++++++++++
 
 Chapel launchers generally arrange for environment variables to be
 forwarded to worker processes. However, this strategy is not always
 reliable. The remote system may override some environment variables, and
 some launchers might not correctly forward all environment variables.
 
 
----------------------------------
-_real binary suffix for execution
----------------------------------
+.. _using-udp:
 
-In order to support profiling tools that produce new binaries for the
-launcher to execute, the suffix of the real binary executed by the
-launcher may be changed with the ``CHPL_LAUNCHER_SUFFIX`` environment
-variable. If this variable is unset, the suffix defaults to "_real",
-matching the compiler's output.
+Using the Portable UDP Conduit
```

Comment:
Sure, definitely some aspects of the conduit and launcher choice are tightly coupled, but there are also things that are peculiar to one and don't have anything to do with the other. And there are other topics (Cray X*) where we've made a choice to push them into a subsidiary file rather than have them up in the main one, and someone else might have decided it the other way.  But we've made a conscious choice to make the top-level files as general as possible, to try to get folks up and running with a minimum of config settings. So since the UDP conduit is our portable default in so many situations I think it's best that that at least the conduit-specific aspects of using it ought to be discussed in multilocale.rst.